### PR TITLE
[PR] Data retrieval work

### PR DESCRIPTION
**Pull request by philcable**
This got a little hairy - hopefully it's not too bad.

Initial work to populate a single profile with data from people.wsu.edu. It doesn't address storing photos - I'm not sure how best to handle that just yet - and raises the question of whether we should provide the Classification terms when the plugin is enabled.

Another question, prompted by the last two commits, is whether a manual refresh on a non-people.wsu.edu site should update just the "Position and Contact Information" meta or all custom meta. This PR updates it all - but more work is needed in order to do it right.

### CODE REVIEW

**Comment by philcable** [comment]
@jeremyfelt Please feel free to start reviewing this whenever you want. I have another small commit I can push to trigger the Travis build once it starts behaving again :)

**Comment by philcable** [comment]
Looks like the Travis build will start working when it's good and ready. @jeremyfelt #reviewmerge?

File: includes/class-wsuwp-people-post-type.php [hunk 1]
```diff
@@ -337,6 +337,7 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 			$profile_vars = array(
 				'nid_nonce' => wp_create_nonce( 'wsu-people-nid-lookup' ),
 				'post_id' => $post->ID,
+				'request_from' => ( apply_filters( 'wsuwp_people_display', true ) ) ? 'people' : 'ad',
```
Comment (jeremyfelt) [inline includes/class-wsuwp-people-post-type.php]: What do you think about this being something like `apply_filters( 'wsuwp_people_source', 'people' );` instead? I can see wanting to reuse a bunch of the front end stuff, so I'm not sure if tying this to whether the display functionality is turned on makes sense.

I'm also wondering if `people` / `ad` is right, or if it should be something like `rest` / `ad`.

Comment (philcable) [inline includes/class-wsuwp-people-post-type.php]: Those changes both make sense to me!

**Comment by philcable** [comment]
Sounds like a plan! I did get stuck in the weeds for a while thinking about how much - if any - should be stored locally. It'll be good to talk through all this!

**Comment by philcable** [comment]
@jeremyfelt I made a couple small steps toward the direction we talked through yesterday.

This preserves the capacity for adding people to a site individually - I think we still want that - but now a majority of a person's data is loaded from people.wsu.edu when their profile is edited. The data that is stored in a local record is (so far) as follows:
* Post title
* NID (leveraged for making the REST request)
* Post content/personal bio (only because I haven't thought through a way to not save it yet :)
* Taxonomy terms (same as above, though it may be beneficial to continue storing them - I just worry about polluting a site's tags...)

I'll rework the front end pieces in another branch! #reviewmerge?

**Pull request merged**